ops/demo-deploy.tmpl.yaml

```
@@ -204,14 +207,35 @@ workerAuxiliary:
     requests:
       memory: "4Gi"
       cpu: "2"
-  extraEnvVars:
-    - name: AUXILIARY_WORKER_THREAD_COUNT
-      value: "1"
-    - name: SIDEKIQ_CONFIG
-      value: "config/sidekiq_auxiliary.yml"
+  extraEnvVars: *envVars
+  extraEnvFrom:
+    - configMapRef:
+        name: auxilary-worker-env
```

I hadn't put this back into pals yet, but in adventist we established this pattern: 

1. Use "aux" instead of "auxiliary" 
2. Put "worker" before "aux" 

```suggestion
        name: worker-aux-env
```